Sources/ScipioKit/Producer/PIF/XCBuildClient.swift

```diff
@@ -134,21 +134,23 @@ struct XCBuildClient {
         try await executor.execute(arguments)
     }
 
-    private func buildCreateXCFrameworkArguments(sdks: Set<SDK>, debugSymbols: [AbsolutePath]?, outputPath: AbsolutePath) throws -> [String] {
-        let frameworksArguments: [String] = try sdks.reduce([]) { arguments, sdk in
+    private func buildCreateXCFrameworkArguments(sdks: Set<SDK>, debugSymbols: [SDK: [AbsolutePath]]?, outputPath: AbsolutePath) throws -> [String] {
```

Comment:
We can reduce this optional since there is no case to get optional value. 

This line can be replaced with `[:]` https://github.com/giginet/Scipio/pull/103/files#diff-6563a8c150e9b20c71a650a47eaaf066cb2d0f52637e56ae9db10bee68e587b4R101

Reply:
@giginet told me that there are no debugSymbols cases so nil is better. Please don't care this review